src/KBarResults.tsx

```diff
@@ -59,8 +61,9 @@ export const KBarResults: React.FC<KBarResultsProps> = (props) => {
       ) {
         event.preventDefault();
         query.setActiveIndex((index) => {
+          const target = continuous ? 0 : index;
```

Comment:
If the first item is not part of a group; e.g. the theme action is selected, this logic wouldn't work:

<img width="778" alt="Screen Shot 2021-12-25 at 3 43 45 PM" src="https://user-images.githubusercontent.com/12195101/147395590-c42bad7d-ca7a-4116-bb83-b4b7835944cb.png">
`

Reply:
@timc1 thanks, i will add an issue.